src/hooks/useQuery.ts

```diff
@@ -38,6 +39,8 @@ export const useQuery = <T = any, V = object>(
 
   const executeQuery = useCallback(
     (opts?: Partial<OperationContext>) => {
+      if (args.skip) return;
```

Comment:
Same as the component, we'll still have to call unsubscribe. It might also make sense to set fetching to false when this happens